# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: media/libcubeb/update.sh
# Commit: f980191897a2
# Full Hash: f980191897a2ceb0514f83df997dc05a349f54db
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2020-01-15 09:38:07
# Description:
#   Bug 1604117 - Backout cubeb commit aa63601. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D59674
# ==============================================================================

diff -r 2ee6c4c25052 -r f980191897a2 media/libcubeb/update.sh
--- a/media/libcubeb/update.sh	Wed Jan 15 01:27:10 2020 +0200
+++ b/media/libcubeb/update.sh	Mon Jan 13 13:09:39 2020 +0000
@@ -91,3 +91,5 @@
 patch -p3 < disable-assert.patch
 echo "Applying disable-iaudioclient3.patch on top of $rev"
 patch -p3 < disable-iaudioclient3.patch
+echo "Applying revert-optimization.patch on top of $rev"
+patch -p1 < revert-optimization.patch
